src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3856,6 +3856,11 @@ public override SyntaxNode ModuloExpression(SyntaxNode left, SyntaxNode right)
             return CreateBinaryExpression(SyntaxKind.ModuloExpression, left, right);
         }
 
+        public SyntaxNode RangeExpression(SyntaxNode left, SyntaxNode right)
+        {
+            return CreateBinaryExpression(SyntaxKind.RangeExpression, left, right);
```

Comment:
Note: we'll likely need formatting support to explicitly remove whitespace around this operator.